### DISCUSSION TITLE
Long polling implementation

### DISCUSSION THREAD

**Discussion by Krischan101** [miguelgrinberg/microdot#208] [Q&A]
Hello everyone, 
I have a quick question, how do you implement long polling with microdot?

**Comment by miguelgrinberg** [comment]
See the implementation I've written for [server-sent events (SSE)](https://github.com/miguelgrinberg/microdot/blob/main/src/microdot/sse.py) to have an idea of how a long-polling solution might be implemented. The sending part would be very similar to that, but SSE does not have a receive side, so that would have to be added.
